sys/net/network_layer/fib/fib.c

```diff
@@ -955,11 +941,6 @@ int fib_sr_entry_add(fib_table_t *table, fib_sr_t *fib_sr,
         return -EFAULT;
     }
 
-    if (fib_sr_check_lifetime(fib_sr, &now) == -ENOENT) {
```

Comment:
I totally agree that the FIB documentation is weak and not helpful.
I will take care of it :)